client/src/components/pages/__tests__/MainChannel.test.tsx

```diff
@@ -56,19 +56,35 @@ describe('[Channel] screen', () => {
     expect(json).toBeTruthy();
     expect(json).toMatchSnapshot();
   });
+});
 
-  describe('interactions', () => {
-    it('should simulate onPress', async () => {
-      const component = createTestElement(<MainChannel />, {
-        environment: mockEnvironment,
-      });
+describe('interactions', () => {
+  it('should simulate [ChanneItem] onPress', async () => {
+    const component = createTestElement(<MainChannel />, {
+      environment: mockEnvironment,
+    });
 
-      const screen = render(component);
-      const btn = await screen.findByTestId('list-item-0');
+    const screen = render(component);
+    const channelItemBtn = await screen.findByTestId('list-item-0');
```

Comment:
I'm afraid that I can't find a good way, "Hackatalk" is not rendered.

Indeed we can give the `accessibilityLabel` to test component instead of `testID`.
But I can't decide whether using accessibilityLabel only for testing is good or bad.

Reply:
I think  adding accessibility label on any icon-only button is a good idea. It helps people who use screen reader.